README.mediawiki

```diff
@@ -981,6 +981,13 @@ Those proposing changes should consider that ultimately consent may rest with th
 | Standard
 | Draft
 |-
+| [[bip-0321.mediawiki|321]]
+| Applications
+| Updated URI Scheme
```

Comment:
This must match the title of the BIP, which currently reads "URI Scheme". One or the other should be adjusted so that they match.